src/home/WalletHome.tsx

```diff
@@ -179,6 +155,39 @@ function WalletHome() {
     renderItem: () => <TransactionFeed key={'TransactionList'} />,
   })
 
+  // Cash In Bottom Sheet
+  const { cashInBottomSheetEnabled } = getExperimentParams(
+    ExperimentConfigs[StatsigExperiments.CHOOSE_YOUR_ADVENTURE]
+  )
+  const hasStable = !!coreTokenBalances.find(
+    (token) => token.balance.gte(STABLE_TRANSACTION_MIN_AMOUNT) && token.address !== celoAddress
+  )
+  const hasCelo =
+    coreTokenBalances
+      .find((token) => token.address === celoAddress)
+      ?.balance.isGreaterThan(CELO_TRANSACTION_MIN_AMOUNT) ?? false
+  const isAccountBalanceZero = hasStable === false && hasCelo === false
+
+  const WrappedCashInBottomSheet = useMemo(() => {
+    if (!isFocused) {
+      return null
```

Comment:
I think we also want to ensure that the bottom sheet will only appear _a single time_ per app session; that is, I think the semantics should be:

* Show the bottom sheet _once_ per app session, on the _first occurrence_ of the following conditions:
  * Bottom sheet is enabled
  * User has zero balance
  * Home screen is focused
 
Namely, we want to avoid showing the bottom sheet multiple times in the situation where a user goes from zero balance -> nonzero balance -> zero balance, all within the same app session.